test/integration/twilio.js

```diff
@@ -7,3 +7,4 @@ try {
   }
 }
 
+
```

Comment:
The only code change is a newline? I thought you had to change the chunkFiles?

Reply:
Nicely spotted, yes that shouldn't be there.